ocaml/xapi/xapi_globs.ml

```diff
@@ -1388,6 +1392,11 @@ let other_options =
     , (fun () -> string_of_bool !ignore_vtpm_unimplemented)
     , "Do not raise errors on use-cases where VTPM codepaths are not finished."
     )
+  ; ( "override-uefi-certs"
+    , Arg.Set override_uefi_certs
+    , (fun () -> string_of_bool !override_uefi_certs)
+    , "Enable/Disable overriding location for varstored UEFI certificates"
```

Comment:
Should this include a hint for the expected value like `Enable (true) or disable (false) overriding ..`?

Reply:
done in the latest fixup